### DISCUSSION TITLE
I can not set protected property in livewire mount method

### DISCUSSION THREAD

**Discussion by sohailahmad07** [livewire/livewire#5620] [1. Help] [answered]
Hi i can not set a protected property in livewire mount method 
 this is my complete code i have strip some for simplicity like the form fields 
 
 Event index blade
```
 @foreach($events as $item)
// show the event here 
<x-model>
  <x-slot:trigger>
Update
</x-slot:trigger>
  <livewire:todos.update event="$item"
       wire:key="{{ $item->getId() }}"/>
</x-model>
@endforeach
```

Event Update blade
```
<div>
  <form wire:submit.prevent="update">
    // form field here
     <x-button>Add Event</x-button>
   </form>

</div>
```

Event Update Compoent

```  

protected $todoEvent;

    public function mount($event)
    {
        $this->todoEvent = $event;


    }

    public function update()
    {
        dd($this->todoEvent);
    }

```

**Comment by joshhanley** [comment]
@sohailahmad07 protected properties aren't persisted to the front end, so you need to make your property public. See the warning here for more details https://laravel-livewire.com/docs/2.x/properties#important-notes

Hope this helps!

**Reply by sohailahmad07** [reply]
@joshhanley I don't want access them on the frontend i just need to access it in the class update method that why i set it to protected

**Reply by ramonmalcolm10** [reply]
@sohailahmad07 protected property value are not persisted between calls from the frontend and the backend. Protected properties values will get disposed upon rendering

**Reply by ramonmalcolm10** [reply]
Public properties are pass back and forth from the frontend to the backend hence why those values can be persisted @sohailahmad07

**Comment by joshhanley** [comment, marked as answer]
@sohailahmad07 yeah @ramonmalcolm10 is correct. You could store the ID of your todoEvent in a public property and then use a computed property to get that from the database for you, if you don't want it to be sent to the front end.
